python/py-blosc2/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-blosc2
+version             2.2.0
+revision            0
+categories-append   devel
+license             BSD
+
+python.versions     37 38 39 310 311
```

Comment:
it [looks](https://github.com/Blosc/python-blosc2/blob/043eab4d63d97efa6b43d95b6876b2b5914a96fe/setup.py#L55) like that Python 3.7 is not supported?

Reply:
Indeed, officially Python 3.7 is not supported, so I removed it (anyway the end of life of Python 3.7 is in June).